gpflow/kernels/mo_kernels.py

```diff
@@ -164,6 +180,11 @@ def __init__(self, kernels, W, name=None):
     def num_latents(self):
         return self.W.shape[-1]  # L
 
+    @property
+    def kernels(self):
+        """The underlying kernels in the multioutput kernel"""
+        return tuple(self._kernels)
+
```

Comment:
Again, is this needed? it should be provided by the Combination inheritance already, no?